misc/ci-builder/Dockerfile

```diff
@@ -10,9 +10,11 @@
 FROM ubuntu:xenial
 
 RUN apt-get update && apt-get install -y --no-install-recommends \
+    bsdmainutils \
```

Comment:
what's this needed for?

Reply:
`dc.sh` depends on the `column` utility, which requires this package :D 

(This wasn't happening before because we were dangerously running dc.sh outside of the ci-builder image, and `column` just happened to be available on the Buildkite agents directly.)

Reply:
Ah nice I look forward to adding most of this functionality to mzcompose so that we can destroy dc.sh and the bash hell that it is.